gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        connection = bq_connection.types.Connection(
</pre><p>Can these classes be initialized by parameters instead of dicts? It's easier for maintenence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>